.github/workflows/build_v2_docker_image.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Publish Docker image
```

Comment:
Please see #68 which breaks this docker image build. Our dockerfile no longer actually copies the full quickannotator package into the container. Instead, the code is mounted into the container from the local directory, as specified in the `devcontainer.json` file.

If you want to build a CI for quickannotator, I would recommend writing a second dockerfile which copies in the code. However, we might want to hold off altogether since we may end up using a docker compose file instead. Thoughts?